boards/v2/phytec/phycore_am62x_a53/Kconfig.phycore_am62x_a53

```diff
@@ -2,5 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_PHYCORE_AM62X_A53
-	bool "PHYTEC phyCORE-AM62x A53"
-	depends on SOC_SERIES_AM6X_A53
+	bool
+	default y
+	select SOC_SERIES_AM6X_A53
```

Comment:
the SoC selects the same series, so this is not needed.
```suggestion
```

Reply:
Fixed